### PR TITLE
Complete website for Math-Magicians pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
- Created a complete site using Routers for navigation.

### CODE REVIEW

File: src/components/pages/CalculatorPage.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Calculator from '../Calculator';
+
+const CalculatorPage = () => (
+  <>
+    <h3>Let Us Do Some Math!</h3>
+    <Calculator />
+  </>
+);
```
Comment (abdulhamiid) [inline src/components/pages/CalculatorPage.js]: Your design
image

Wireframe
image

As it is important that our general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  👀


Kindly adjust the layout of the text `Let Us Do Some Math!` and `Calculator` body according to the wireframe. 🛠👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @AleWaweru 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use React Router.
- ✔️ Use styles in a React app.


## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/pages/CalculatorPage.js (hunk 1 above)
Comment (AleWaweru) [reply src/components/pages/CalculatorPage.js]: done.

**Review by levy002 — APPROVED** [review]
## Hello @AleWaweru  
 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

  unnamed


## Highlights
- You implemented all suggested changes :heavy_check_mark: 
 
## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**